qsub_mosaicSubTiles.sh

```diff
@@ -73,15 +74,15 @@ export BWPY_PREFIX=""
 if [ "${p9}" == 'null' ]; then
     echo "Quad arg not present. Running full tile"
 
-    echo matlab -nojvm -nodisplay -nosplash -r "try; addpath('${p1}'); addpath('${p2}'); [x0,x1,y0,y1]=getTileExtents('${p7}','${p8}'); disp(x0); ${p3}('${p4}',${p5},'${p6}','projection','${p11}','version','${p12}','extent',[x0,x1,y0,y1]); catch e; disp(getReport(e)); exit(1); end; exit(0);"
-    time matlab -nojvm -nodisplay -nosplash -r "try; addpath('${p1}'); addpath('${p2}'); [x0,x1,y0,y1]=getTileExtents('${p7}','${p8}'); disp(x0); ${p3}('${p4}',${p5},'${p6}','projection','${p11}','version','${p12}','extent',[x0,x1,y0,y1]); catch e; disp(getReport(e)); exit(1); end; exit(0);"
+    echo matlab -nodisplay -nosplash -r "try; addpath('${p1}'); addpath('${p2}'); warning('off','all'); parpool($CORES_PER_NODE); [x0,x1,y0,y1]=getTileExtents('${p7}','${p8}'); disp(x0); ${p3}('${p4}',${p5},'${p6}','projection','${p11}','version','${p12}','extent',[x0,x1,y0,y1]); catch e; disp(getReport(e)); exit(1); end; exit(0);"
+    time matlab -nodisplay -nosplash -r "try; addpath('${p1}'); addpath('${p2}'); warning('off','all'); parpool($CORES_PER_NODE); [x0,x1,y0,y1]=getTileExtents('${p7}','${p8}'); disp(x0); ${p3}('${p4}',${p5},'${p6}','projection','${p11}','version','${p12}','extent',[x0,x1,y0,y1]); catch e; disp(getReport(e)); exit(1); end; exit(0);"
     task_return_code=$?
 
 else
     echo "Running quadrant ${p9}"
 
-    echo matlab -nojvm -nodisplay -nosplash -r "try; addpath('${p1}'); addpath('${p2}'); [x0,x1,y0,y1]=getTileExtents('${p7}','${p8}','quadrant','${p9}'); ${p3}('${p4}',${p5},'${p6}','projection','${p11}','version','${p12}','quadrant','${p9}','extent',[x0,x1,y0,y1]); catch e; disp(getReport(e)); exit(1); end; exit(0);"
-    time matlab -nojvm -nodisplay -nosplash -r "try; addpath('${p1}'); addpath('${p2}'); [x0,x1,y0,y1]=getTileExtents('${p7}','${p8}','quadrant','${p9}'); ${p3}('${p4}',${p5},'${p6}','projection','${p11}','version','${p12}','quadrant','${p9}','extent',[x0,x1,y0,y1]); catch e; disp(getReport(e)); exit(1); end; exit(0);"
+    echo matlab -nodisplay -nosplash -r "try; addpath('${p1}'); addpath('${p2}'); warning('off','all'); parpool($CORES_PER_NODE); [x0,x1,y0,y1]=getTileExtents('${p7}','${p8}','quadrant','${p9}'); ${p3}('${p4}',${p5},'${p6}','projection','${p11}','version','${p12}','quadrant','${p9}','extent',[x0,x1,y0,y1]); catch e; disp(getReport(e)); exit(1); end; exit(0);"
+    time matlab -nodisplay -nosplash -r "try; addpath('${p1}'); addpath('${p2}'); warning('off','all'); parpool($CORES_PER_NODE); [x0,x1,y0,y1]=getTileExtents('${p7}','${p8}','quadrant','${p9}'); ${p3}('${p4}',${p5},'${p6}','projection','${p11}','version','${p12}','quadrant','${p9}','extent',[x0,x1,y0,y1]); catch e; disp(getReport(e)); exit(1); end; exit(0);"
     task_return_code=$?
```

Comment:
@khsa1 I'm sure you noticed that the structure of `qsub_mosaicSubTiles.sh` and the corresponding `batch_mosaicSubTiles.py` Python wrapper script are quite different from the buildSubTiles counterparts. I have yet to get the mosaicSubTiles wrappers refactored to function like the buildSubTiles ones on Blue Waters. But I'm glad to see you were able to get MST to work without too much effort apparently!

Reply:
@ehusby I've only tested this so far on our systems at OSC but I think the changes should be minimal to get it working on Bluewaters. I can add it to this merge request if I get it working.

Reply:
@khsa1 Okay. Do note that I've made some considerable organizational changes on BW to the location of source data and folder structure of the results. It might be simplest to wait until I've had a chance to merge changes to the default filepaths in the `batch_buildSubTiles.py` wrapper script and corresponding `batch_buildSubTiles_bw.patch` file.